core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -403,6 +403,8 @@ User.IllegalFullname="{0}" is prohibited as a full name for security reasons.
 
 TimeZoneProperty.DisplayName=User Defined Time Zone 
 TimeZoneProperty.DisplayDefaultTimeZone=Default
+TimeZoneProperty.current_time_in_=Current time in {0}: {1}
+TimeZoneProperty.current_time_on_server_in_in_proposed_di=Current time on server in {0}: {1}; in proposed display zone: {2}
```

Comment:
what's di referring to here? 'proposed_di'

Reply:
It says `in` here twice, as well as in the code

Reply:
Just abbreviations of the English text, generated with an IDE plugin.